src/main.rs

```diff
@@ -25,12 +26,40 @@ arg_enum! {
     }
 }
 
+/// Identify if file is sass entrypoint.
```

Comment:
I try to maintain uniform casing for Sass and SCSS in documentation and comments. Could you update this comment and the one immediately below it to reflect the preferred casing